go.yml

```diff
@@ -0,0 +1,56 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: "0 10 * * 1-5"
```

Comment:
I think we don't need cron for CI. We need add cron in another action file to build and push image.

Reply:
The goal is to run multiple times a day just to make sure there are no false positives. I need to tune the cron schedule though.

Reply:
👍 OK